lib/pymice/_Ens.py

```diff
@@ -22,8 +22,11 @@
 #    along with this software.  If not, see http://www.gnu.org/licenses/.     #
 #                                                                             #
 ###############################################################################
-
-import collections
+import sys
+if sys.version_info >= (3, 3):
+  import collections.abc as collections
```

Comment:
Since we do not test/support Python < 3.3 (c027098, 9f89ae9) I think the conditional import is unnecessary and adds more code to be removed sometime in the future.
I think `import collections.abc` should be enough (appropriate change in line 215 is necessary).  Same applies to other files.